package.json

```diff
@@ -1,38 +1,30 @@
 {
-  "name": "movies-search",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^11.2.7",
-    "@testing-library/user-event": "^12.8.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
-    "web-vitals": "^1.1.2"
-  },
+  "name": "hometask2",
+  "version": "1.0.0",
+  "description": "[<img alt=\"Deployed with FTP Deploy Action\" src=\"https://img.shields.io/badge/Deployed With-FTP DEPLOY ACTION-%3CCOLOR%3E?style=for-the-badge&color=2b9348\">](https://github.com/SamKirkland/FTP-Deploy-Action)  [<img alt=\"Hosted through 000webhost\" src=\"https://img.shields.io/badge/Hosted through-000webhost-%3CCOLOR%3E?style=for-the-badge&color=red\">](https://in.000webhost.com/)",
+  "main": "webpack.config.js",
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "webpack-dev-server --mode development --open --hot",
+    "build": "webpack --mode production"
```

Comment:
Please refer [https://create-react-app.dev/docs/adding-custom-environment-variables/](https://create-react-app.dev/docs/adding-custom-environment-variables/) to using environment variables